src/incremental/compareCFG.ml

```diff
@@ -45,7 +42,7 @@ module NTH = Hashtbl.Make(
  * comparison is done for its successors. The two function entry nodes make up the tuple to start the comparison from. *)
 let compareCfgs (module CfgOld : CfgForward) (module CfgNew : CfgForward) fun1 fun2 =
   let diff = NH.create 113 in
-  let same = NTH.create 113 in
+  let same = ref {node1to2=NH.create 113; node2to1=NH.create 113} in
```

Comment:
Does this need to be a `ref` at all? Both the hashtables themselves are mutable anyway.